src/topp/CometAdapter.cpp

```diff
@@ -209,6 +209,15 @@ class TOPPCometAdapter :
     registerIntOption_("spectrum_batch_size", "<posnum>", 20000, "max. number of spectra to search at a time; use 0 to search the entire scan range in one batch", false, true);
     setMinInt_("spectrum_batch_size", 0);
     registerDoubleList_("mass_offsets", "<doubleoffset1, doubleoffset2,...>", {0.0}, "One or more mass offsets to search (values subtracted from deconvoluted precursor mass). Has to include 0.0 if you want the default mass to be searched.", false, true);
+    registerStringOption_("pinfile_protein_delimiter", "<delimiter>", "\t", "specify a different character or string for the protein column delimiter (default tab)",false, false);
```

Comment:
definitely an advanced option
```suggestion
    registerStringOption_("pinfile_protein_delimiter", "<delimiter>", "\t", "specify a different character or string for the protein column delimiter (default tab)", false, true);
```